scripts/aws/setupdb.sh

```diff
@@ -69,7 +69,10 @@ envdir /etc/mmw.d/env /opt/app/manage.py migrate
 
 function download_and_load {
     for f in "${FILES[@]}"; do
-        curl -s $FILE_HOST/$f | gunzip -q | psql --single-transaction
+        rm -f tmp.sql tmp.sql.gz
```

Comment:
Okay, that makes sense.

Can we make it something like:

```bash
if [ -ne $f.sql ]; then
    curl ... && gunzip ... && psql .. && rm -f $f.sql $f.sql.gz
fi;
```

so that in case there's an error in one of these steps, the file doesn't have to be redownloaded the next time?